doc/man3/OSSL_CMP_CTX_snprint_PKIStatus.pod

```diff
@@ -0,0 +1,46 @@
+=pod
+
+=head1 NAME
+
+OSSL_CMP_CTX_snprint_PKIStatus
+- function(s) for managing the CMP PKIStatus
+
+=head1 SYNOPSIS
+
+ #include <openssl/cmp.h>
+
+ char *OSSL_CMP_CTX_snprint_PKIStatus(OSSL_CMP_CTX *ctx, char *buf, int bufsize);
```

Comment:
snprintf functions return a len, not the pointer received as input arg.
e.g.: http://www.cplusplus.com/reference/cstdio/snprintf/ 
I feel it a bit weird, but you can keep it as this.

Reply:
I understand. But changing the return style would affect a lot of places in our code and documentation. So I will keep it as it is.